src/buttons.ts

```diff
@@ -0,0 +1,63 @@
+// LCD game JavaScript library
+// Bas de Reuver (c)2018
+
+import { Button, ButtonType } from "./@types";
```

Comment:
File contains helper methods to deal with the structure of the gameconfig.json in an easier way.